src/Microsoft.VisualStudio.Services.Agent/ConfigurationStore.cs

```diff
@@ -59,19 +59,33 @@ public sealed class AgentSettings
         public string CollectionName { get; set; }
     }
 
+    [DataContract]
+    public sealed class AutoLogonSettings
```

Comment:
IConfigurationStore is dependent on this and we are consuming IConfigurationStore in AutoLogonManager as well. I think it will be good to have it to avoid the circular dependency.

---
In reply to: [119687649](https://github.com/Microsoft/vsts-agent/pull/985#discussion_r119687649) [](ancestors = 119687649)